compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -401,6 +399,26 @@ void addConvTileAndDecomposeExpertPassPipeline(OpPassManager &passManager) {
     nestedModulePM.addNestedPass<func::FuncOp>(createCSEPass());
   }
 
+  // Add the sandbox single tiling expert to vectorize.
+  // We can't do the vectorization in the tiling expert above due to an issue in
+  // codegen strategy pipeline. Since we are moving to the transform dialect, we
+  // choose to have a workaround here by splitting them into two stages.
+  {
+    LinalgSingleTilingExpertPassOptions options;
```

Comment:
How do we know this doesnt tile again with the first level of tiling.....

Reply:
By default `tilingLevel = -1` and `tileSizes = []` in `LinalgSingleTilingExpertPassOptions`. Therefore the tile transformation in LinalgSingleTilingExpertPass will be disabled. It seems to be confusing tho...

Reply:
The ops won't get tiled if `tilingLevel` is not specified.